src/angular/notification-toast/notification-toast-container.scss

```diff
@@ -35,34 +35,35 @@
 }
 
 .sbb-notification-toast-error {
-  background-color: var(--sbb-color-error);
+  background-color: var(--sbb-notification-toast-background-color-error);
 
   :is(&, & :is(a, button)) {
-    color: var(--sbb-color-white);
+    color: var(--sbb-notification-toast-label-color-error);
   }
 }
 
 .sbb-notification-toast-success {
-  border: var(--sbb-border-width-thin) solid var(--sbb-color-green);
-
+  border: var(--sbb-border-width-thin) solid var(--sbb-notification-toast-label-color-success);
```

Comment:
The border color for success notification-toasts should be green in dark mode. I think we need another property `--sbb-notification-toast-border-color-{success,error,information,warn}`